scripts/xcode-test.sh

```diff
@@ -14,7 +14,7 @@ REF_NAME="${3-HEAD}"
 IS_LOCAL_BUILD="${4:-ci}"
 COMMAND="${5:-test}"
 DEVICE=${6:-iPhone 14}
-CONFIGURATION_OVERRIDE="${7}"
+CONFIGURATION_OVERRIDE="${7:-}"
```

Comment:
m: Why do we need this change?

Reply:
This is technically unrelated, but right now calling `make test` doesn't work because it doesn't find this environment property. I'm guessing it's mostly being run from CI which sets the property and it was an oversight to not include this as optional for local runs

Reply:
thanks for catching and correcting this 🙏🏻